docs/tutorials/transforms.ipynb

```diff
@@ -500,7 +533,11 @@
     "id": "w4ZbjxPyHoiB"
    },
    "source": [
-    "It's even possible to chain indices along with augmentations from Kornia for a single callable during training."
+    "It's even possible to chain indices along with augmentations from Kornia for a single callable during training.\n",
+    "\n",
+    "Check out the documentation of Kornia's augmentations [here](https://kornia.readthedocs.io/en/latest/augmentation.html).\n",
```

Comment:
Let's add an Additional Reading section at the end like I did in my Intro tutorials